src/Tizen.Multimedia.MediaPlayer/Player/EqualizerBand.cs

```diff
@@ -56,12 +48,16 @@ internal EqualizerBand(AudioEffect owner, int index)
         /// <exception cref="ArgumentOutOfRangeException">
         ///     <paramref name="value"/> is not inside of <see cref="AudioEffect.BandLevelRange"/>.
         /// </exception>
+        /// <exception cref="InvalidOperationException">
+        ///     If audio offload is enabled by calling <see cref="AudioOffload.IsEnabled"/>. (Since tizen 6.0)
+        /// </exception>
         /// <since_tizen> 3 </since_tizen>
         public int Level
         {
             get
             {
                 _owner.Player.ValidateNotDisposed();
+                _owner.Player.AudioOffload.CheckDisabled();
```

Comment:
Why do to check disabling? it seem to no need to check disable becuase it just get

Reply:
it is related to an internal policy of player. that is added just for a double check.